skin-schema.json

```diff
@@ -716,6 +716,36 @@
         }
       }
     },
+    "isVrAnimationEnabled": {
```

Comment:
I think these properties (`isVrAnimationEnabled`, `animationDuration`) should be inside a `vr` (or similarly named) section.

Reply:
Also, I think `isVrAnimationEnabled` sound like it should be a boolean value. Should this be something like `vrAnimation` and then have a separate `enabled` property?

Reply:
Oh ok, if this was already there then I think we can keep it.